bitnami/redis/templates/master/statefulset.yaml

```diff
@@ -290,6 +290,11 @@ spec:
               mountPath: /opt/bitnami/redis/certs
               readOnly: true
             {{- end }}
+            {{- if .Values.metrics.extraArgs.script }}
+            - name: {{ printf "%s-metrics-script-file" (include "common.names.fullname" .) }}
+              mountPath: {{ printf "/mnt/%s/" (include "common.names.name" .) }}
+              readOnly: true
+            {{- end }}
```

Comment:
Instead of this, we should be mounting `metrics.extraVolumeMounts` the same way we are mounting `master.extraVolumeMounts`.

Reply:
That does align the code with the docs I wrote for it.  Whoops.

This does somewhat alter the intent of the change, however.  Now it allows for effectively ad-hoc volume mounts into the metrics sidecar, and using that to setup the custom script is basically a side-effect.  I consider this a Good Thing™.